content/en/docs/setup/production-environment/_index.md

```diff
@@ -9,7 +9,7 @@ no_list: true
 A production-quality Kubernetes cluster requires planning and preparation.
 If your Kubernetes cluster is to run critical workloads, it must be configured to be resilient.
 This page explains steps you can take to set up a production-ready cluster,
-or to uprate an existing cluster for production use.
+or to upgrade an existing cluster for production use.
```

Comment:
```suggestion
or to promote an existing cluster for production use.
```
I like Brad's suggestion. What do you think @tmeralus ?

Reply:
That is more clear 👌🏾 I like that

Reply:
@tmeralus Feel free to commit the change on your end